.github/workflows/publish_develop.yml

```diff
@@ -25,8 +25,8 @@ on:
     branches:
       - develop
     paths-ignore:
-      - 'CHANGELOG.md'
-      - 'VERSION'
+      - "CHANGELOG.md"
```

Comment:
does single vs double quote matter in the actions?

Reply:
It does if you're doing variable expansion or using some related shell commands, but it doesn't matter here. This is enforcing the precommit rule.